### PR TITLE
DOCS-192 Add Cosmos Staking guide

### PR DESCRIPTION
**Pull request by pyr0gan**

### CODE REVIEW

File: docs/guides/staking-api/Cosmos/staking/4_submit-signed-transaction.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "transaction_payload": "0aa4010a8f010a1c2f636f736d6f732e62616e6b2e763162657461312e4d736753656e64126f0a2d636f736d6f73313738366a6a6b6835786b343376636d72633439307a6e34386668386e6c303274637572306365122d636f736d6f73313561707464716d6d376464677463726a7663356873393838726c726b7a6534306c34713068651a0f0a057561746f6d1206313030303030121054657374696e67206175746f2d67617312670a500a460a1f2f636f736d6f732e63727970746f2e736563703235366b312e5075624b657912230a210269bc2f2b61cdddb0bb2cf169d5fad0afc19326aa13d2bf967ee3092f7ade88f712040a020801181d12130a0d0a057561746f6d12043138343310d1bf041a40a987ccc429bbeca8f8e3295f3ccb800a35400944ca17801ee888cfb28f1ec6b40d654a45aedfc46f3e1f825951ace283f911258131a989d925827bd322b2b0fb"
+  }
+}'
+```
+
+### Response
+
+With the signed transaction provided, the Staking API will broadcast it to the network. As the transaction progresses towards confirmation, the returned state will be either:
+
+- `transfer_tx_broadcasting` &mdash; Transaction has been broadcast and is waiting for confirmation.
```
Comment (yash-sharma1) [inline docs/guides/staking-api/Cosmos/staking/4_submit-signed-transaction.mdx:33]: I doubt the state mentioned here is incorrect, this state of transaction belongs to Transfer flow, it should be `delegate_tx_broadcasting`

File: docs/guides/staking-api/Cosmos/staking/5_get-flow-status.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+-H 'Authorization: <api_key>'
+```
+
+### Response
+
+- `state` &mdash; `string` The current state of the flow.
+
+:::info note
+
+The `state` field will update from `transfer_tx_broadcasting` to `completed`.
```
Comment (yash-sharma1) [inline docs/guides/staking-api/Cosmos/staking/5_get-flow-status.mdx:25]: The same as the previous step, this state of transaction belongs to the Transfer flow, it should be `delegate_tx_broadcasting`.

File: docs/guides/staking-api/Cosmos/staking/4_submit-signed-transaction.mdx (hunk 1 above)
Comment (yash-sharma1) [reply docs/guides/staking-api/Cosmos/staking/4_submit-signed-transaction.mdx:33]: yes this `transfer_tx_broadcasting` is incorrect, ref to sample response line 3rd

**Review by yash-sharma1 — CHANGES_REQUESTED** [review]

File: docs/guides/staking-api/Cosmos/staking/index.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: Transfer
+slug: /guides/staking-api/cosmos/stake
+sidebar_position: 1
+---
+
```
[the remark this answers is not in the document]
Comment (pyr0gan) [reply docs/guides/staking-api/Cosmos/staking/index.mdx:6]: This is done, I just had uncommitted changes. Thanks @yash-sharma1!

File: docs/guides/staking-api/Cosmos/staking/2_submit-delegation-data.mdx [hunk 4]
```diff
@@ -0,0 +1,113 @@
+---
+title: Submit Delegation Data
+slug: /guides/staking-api/cosmos/staking/submit-delegation-data
+---
+
+After collecting the required inputs, send a `PUT` request to `/api/v1/flows/[:flow_id]/next` to proceed to the next step.
+
+### Parameters
+
+- `name` &mdash; `string` Name of the action to execute - `create_delegate_tx`.
```
[the remark this answers is not in the document]
Comment (pyr0gan) [reply docs/guides/staking-api/Cosmos/staking/2_submit-delegation-data.mdx:10]: Got both 😄

**Comment by pyr0gan** [comment]
@yash-sharma1 - Requested changes applied!

**Review by yash-sharma1 — APPROVED** [review]

**Pull request merged**